native/cocos/editor-support/spine-wasm/spine-skeleton-instance.cpp

```diff
@@ -420,6 +430,14 @@ void SpineSkeletonInstance::setListener(uint32_t listenerID, uint32_t type) {
     }
 }
 
+void SpineSkeletonInstance::setTrackListener(uint32_t trackId, TrackEntry *entry) {
+    if (!entry->getRendererObject()) {
+        _trackListenerID = trackId;
+        entry->setRendererObject(this);
+        entry->setListener(trackEntryCallback);
+    }
```

Comment:
Should we output an error if render object is invalid?

Reply:
render object only used to prevent setTrackListener multi times

Reply:
So setTrackListener could not be invoked multiple times for the same trackId?
If I want to replace entry for the same trackId, how to achieve that?

Reply:
trackId is generated by engine internal, one trackEntry only has one trackListener. While dispose_event called, trackEntry's listener will be reset to null.